grails-app/views/requisition/_showRequisitionItem.gsp

```diff
@@ -29,10 +29,15 @@
         </td>
     </g:if>
     <td class="middle">
-        <div class="tag ${requisitionItem.isCanceled() ? 'tag-danger' : requisitionItem.isSubstituted() || requisitionItem.isReduced() ? 'tag-warning' : 'tag-alert'}">
+        <div class="tag ${requisitionItem.isCanceled() || requisitionItem.requisition.status >= RequisitionStatus.PICKED &&
```

Comment:
@DominikaHatala Reminder: any time we need complex logic in the UI it would be best to move the logic into the domain class. For example, the following method might be useful on RequisitionItem.
```
boolean isNotPicked() { 
    requisition.status >= RequisitionStatus.PICKED && calculateQuantityPicked() == 0
}
```